src/braket/default_simulator/openqasm/program_context.py

```diff
@@ -751,7 +740,7 @@ def add_builtin_gate(
 
     @abstractmethod
     def add_gate_instruction(
-        self, gate_name: str, target: Tuple[int], params, ctrl_modifiers: List[int], power: int
+        self, gate_name: str, target: Tuple[int, ...], params, ctrl_modifiers: List[int], power: int
```

Comment:
Interesting, how come we haven't been doing this the whole time? I feel like there are a lot of places that would get updated haha

Reply:
While we're at it, are we on a python version where we can finally switch to `tuple[int, ....]` or no? Looks like 3.9+

Reply:
Why we haven't done this the entire time I'll chalk up to it being easy to forget and treat the same as `Liszt`.

We're on Python 3.8, so almost there!

Reply:
Oh does list assume arbitrary length while tuple doesn't? Another note, we're sometimes imprecise about our type annotations. I believe this method will work as long as target is `Iterable[int]`

Reply:
Yeah, `Tuple` serves a weird dual purpose as both a fixed tuple of items (e.g. `Tuple[int, str, float]`) and a variable-length immutable list. It would be nice for these concepts to be separate, but Python...